js/piwik.js

```diff
@@ -1403,7 +1403,17 @@ if (typeof Piwik !== 'object') {
             },
             hasNodeCssClass: function (node, klassName)
             {
-                if (node && klassName && node.className) {
+                if(!node || !klassName) {
+                    return false;
+                }
+
+                // For modern browsers cf http://caniuse.com/#search=classList
+                if(node.classList) {
+                    return node.classList.contains(klassName);
```

Comment:
Do all browsers that implement `node.classList` also return a `DOMTokenList` that includes the `contains()` method?

A problem is that our tests will only cover one of both ways. Possibly we should split the method into two methods and test them separately @mattab ? Or we need to find a way to run these tests in an old browser eg IE6 regularly and on newer ones